packages/forms/backend/sirius-components-forms/src/main/java/org/eclipse/sirius/components/forms/DateTimeStyle.java

```diff
@@ -0,0 +1,116 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.components.forms;
+
+import java.text.MessageFormat;
+import java.util.Objects;
+
+import org.eclipse.sirius.components.annotations.Immutable;
+
+/**
+ * The style of a DateTime.
+ *
+ * @author lfasani
+ */
+@Immutable
+public final class DateTimeStyle extends AbstractFontStyle {
```

Comment:
Why `DateTimeStyle` inherits from `AbstractFontStyle`?

Reply:
good catch ;)
I removed the inheritance